build_wasm.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
```

Comment:
Nitpicking, but... could be a typescript file and be placed in `bin/` folder as well?